Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM gradle:jdk-alpine
+
+USER root
```

Comment:
Is this required for the bash install?

Reply:
I was getting odd permission errors when creating `/opt/analyzer`, so I just set the user as root

Reply:
Weird. It should work, see the [C# analyzer Dockerfile](https://github.com/exercism/csharp-analyzer/blob/master/Dockerfile).

Reply:
I think it may be from the [gradle:jdk-alpine dockerfile](https://github.com/keeganwitt/docker-gradle/blob/0c2624d788edb813667a1e72659f8be612d420f3/jdk8-alpine/Dockerfile#L33)...